subjects/ai/training/audit/README.md

```diff
@@ -16,29 +16,19 @@
 
 #### Exercise 1: MSE Scikit-learn
 
-##### The goal of this exercise is to learn to use `sklearn.metrics` to compute the mean squared error (MSE).
+###### Is the Mean Squared Error (MSE) calculated using `sklearn.metrics` library?
 
-1. Compute the MSE using `sklearn.metrics` on `y_true` and `y_pred` below:
-
-```python
-y_true = [91, 51, 2.5, 2, -5]
-y_pred = [90, 48, 2, 2, -4]
-```
+###### Is the Mean Squared Error (MSE) correctly computed for the given `y_true` and `y_pred` values, and does the calculated MSE match the expected value?
 
 ---
 
 ---
 
 #### Exercise 2: Accuracy Scikit-learn
 
-##### The goal of this exercise is to learn to use `sklearn.metrics` to compute the accuracy.
+###### Is the accuracy computed using `sklearn.metrics` library?
```

Comment:
Article missing.
Is the accuracy computed using **THE** `sklearn.metrics` library?